sql/moz-fx-data-shared-prod/messaging_system_derived/normalized_onboarding_events/view.sql

```diff
@@ -0,0 +1,28 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.messaging_system_derived.normalized_onboarding_events`
+AS
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  submission_timestamp AS timestamp,
+  version AS app_version,
+  message_id AS category,
+  ARRAY(
+    SELECT AS STRUCT key, value
+    FROM UNNEST([
```

Comment:
The `event_properties` come in as a JSON field. I'd love to automatically extract all event properties from that JSON blob, but BQ JSON primitives aren't good for that kind of processing right now, and it's not worth it to expend the energy on some complicated regex expressions.

Instead, we'll pull them out individually; we'll just check-in with the UJET team about:
- which fields they need
- how we want to handle new fields